tests/tests.py

```diff
@@ -160,7 +145,8 @@ def __init__(self):
                         logging=False,
                         max_retries=self.max_retries,
                     )
-            if not self.jira_admin.current_user():
+
+            if not self.jira_admin.current_user("displayName") == "CI Admin":
```

Comment:
I'll find a value that the local tests and cloud tests can agree on that exists for the admin user or otherwise make changes to the tests so that they agree.

Reply:
It might not be such a big issue, there is much more to be configured before an instance spinned up with `atlas-run-standalone` works.

Shouldn't this check be more like "check if the current user is logged in ([optional] with any admin account)", rather than "check if the current user is logged in with an account with exact name XYZ"?

Reply:
From my reading of the original test it was to check whether the call to `myself` works, which shows you are currently logged in. To be honest at this point I don't remember why exactly I used `displayName`, but I do recall I was having issues with getting this test to pass until I checked the username.

So in terms of the original intention of the test, I agree with you. It would be better for this test to pass as long as it returns anything, although I'm assuming it throws an error or otherwise not `200` if you're not logged in. I'll revisit this test.